content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -2,77 +2,77 @@
 Assign leads based on scoring
 =============================
 
-With *Leads Scoring* you can automatically rank your leads based on
-selected criterias.
+With *Lead Scoring* you can automatically rank your leads based on selected criteria.
 
-For example you could score customers from your country higher or the
-ones that visited specific pages on your website.
+For example, you could score customers higher if coming from the same country as your
+company over a customer from a different country.
 
 Configuration
 =============
 
-To use scoring, install the free module *Lead Scoring* under your
-*Apps* page (only available in Odoo Enterprise).
+To use scoring, install the free module *Lead Scoring* under your *Apps* page (only available in
+Odoo Enterprise).
 
-.. image:: media/lead_scoring01.png
+.. image:: lead_scoring/lead-scoring-module.png
    :align: center
+   :alt: Lead Scoring module installation
 
 Create scoring rules
 ====================
 
-You now have a new tab in your *CRM* app called *Leads Management*
-where you can manage your scoring rules.
+You now have a new tab in your *CRM* app called *Leads* where you can manage your scoring rules.
 
-Here's an example for a Canadian lead, you can modify for whatever
-criteria you wish to score your leads on. You can add as many criterias
-as you wish.
+Here's an example of a scoring rule for English-speaking Canadian users. You can modify your rules
+to match whatever criteria you want to score leads on, and you can add as many criteria as you would
+like.
 
-.. image:: media/lead_scoring02.png
+.. image:: lead_scoring/scoring-example.png
```

Comment:
Along these lines, do we have a width that we prefer to stay around in terms of size? If so, could we maybe add that to the documentation style guides?

Reply:
Good point!

Well, so far we used to take screenshots on large 1080p screens and we ended up having good screenshots just like that (with the current guidelines) so we didn't even think of possible other resolutions.
But indeed, now that more and more people contribute to the doc, that's definitely something to add... Especially since it varies according to the screen size as well (that's rather for the "zoom" than the resolution, but still)

But in your case, what was it? Is it a 4K screen? Or what software do you use to take screenshots? I know KC had a "double" resolution as well but I don't remember how he took the screenshots (it seemed rather complex, actually)

Thanks :)

Reply:
Hey! 

So I just checked, and both my laptop and the external monitor are set at 1920x1080. I have windows maximized when I take screenshots. 

I've been using windows+shift+s (or cmd+shift+4) to take a screenshot and save the snippet. If I need to add a box or anything, then I just toss it in paint. Once I add the new media folder to Git, I go in and pngquant all of them.

Reply:
@jcs-odoo I also think [Scaling](https://i.imgur.com/RQKVoN6.png) should be taken into consideration, no?